spec/components/schemas/content/notification-optin.ts

```diff
@@ -0,0 +1,62 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './base';
+import { createComponentRef } from '../../../../utils/ref';
+
+const notificationOptIn: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'object',
+        example: 'notification_optIn',
+      },
+      medialUrl: {
+        description: 'URL for the thumbnail the user will see.',
+        type: 'string',
+        example: 'https://cdn.zenvia.com/main-picture.jpg',
+      },
+      title: {
+        description: 'The topic of the recurring subscription.',
```

Comment:
na descrição não precisa economizar, é onde une tudo, eu colocaria: `The topic of the recurring notification subscription.`

Reply:
@rafael-org show!
E os detalhes como: "Só pode mandar 10 tópicos diferentes em um período de 10 dias". Vai aqui ou em outra parte da documentação?